### DISCUSSION TITLE
blog/2023/03/how-to-fix-an-issue-with-the-obsidian-app-not-opening-and-syncing-on-your-iphone/

### DISCUSSION THREAD

**Discussion by giscus** [josh-wong/josh-wong.github.io#7] [General]
# blog/2023/03/how-to-fix-an-issue-with-the-obsidian-app-not-opening-and-syncing-on-your-iphone/

Ramblings of a technical writer in Tokyo🗼🗾

https://www.080f53.com/blog/2023/03/how-to-fix-an-issue-with-the-obsidian-app-not-opening-and-syncing-on-your-iphone/

**Comment by EricEWeir** [comment]
I’ve tried all your suggestions. Was hopeful about the most recent on. Alas.

My problem is not files not downloading/syncing. One configuration file—.obsidian.vimrc—is not syncing. The configuration works on the desktop app (macOS). Not on the mobile (iOS).

**Reply by EricEWeir** [reply]
Update: Just discovered that the problem is the same on both mobile and desktop: the <esc> remapping doesn’t work; all the others do. Again, on both apps.

**Reply by josh-wong** [reply]
It's a bit of a strange issue, huh? I encountered the issue recently again on a couple of occasions. Seems totally random. 

I've slightly updated the instructions in this post. In particular, I added a part about needing to be connected to Wi-Fi. I couldn't fix the issue when connected to a mobile network.

Unfortunately, I'm not sure how to fix the issue on the desktop side, since I've narrowed down my problem to being a sync issue on the iPhone.

**Comment by CarolusIII** [comment]
Next time, you could try restarting iCloud syncing.
Open te Activities app
Find the activity "bird" and force stop it.
Then do the same to "cloudd".
If it comes back, kill it once more.

This has helped me several times when I had syncing issues.

**Reply by josh-wong** [reply]
Sorry, I’m not really sure what you mean. The instructions you provided don’t make sense in iOS.

**Reply by CarolusIII** [reply]
That's right, but iCloud is connected to your appleID and that also includes your iPhone. The procedure I mentioned can only be initiated on a desktop, but it controls the iCloud sync process that also influences the iPhone as it uses the cloud sync just like the desktop does. Hopefully this is clear now.

**Reply by josh-wong** [reply]
I see. This process might work on macOS, but since I’m on Windows, I can’t confirm.
